packages/vm/scripts/diffTester.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+usage() {
+    echo 'Usage: ./diffTester.sh [-a branch1] [-b branch2] [-t "path/to/my/test.json"] [-r 5] [-f London]'
```

Comment:
Not sure, might this be overengineered to allow to pass in two branches here and might it be enough to just always compare against the current branch, guess that's the main use case (and then call the option maybe just `-b` for branch)?

But totally no strong opinion here and can go either way.

Reply:
Fair point.  Will update accordingly.